CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contributing
+
+Our goal is to make contributing to the `muta` project easy and transparent.
+
+When contributing to this repository, please first discuss the change you wish to make via issue, or any other method with the community before making a change. 
+
+### Report Issue
+
+* Read known issues to see whether the issue is already addressed there.
+
+* Search existing issues to see whether others had already posted a similar issue.
+  
+* **Do not open up a GitHub issue to report security vulnerabilities**. Instead,
+  refer to the [security policy](SECURITY.md).
+
+* When creating a new issue, be sure to include a title and clear description. It is appreciated that if you can also attach as much relevant information as possible, such as version, environment, reproducing steps, samples.
+
+### Send PR
+
+* See [Code Standards]() for code guidelines.
```

Comment:
Where are the links to `Code Standards`?

Reply:
After code standards document is ready, i will fix that